### PR TITLE
Update test with corner case for table

### PR DESCRIPTION
**Pull request by flynneva**
@liran-funaro I believe this is what #3 is about.

Let me know if you have any questions

### CODE REVIEW

File: tests/source/ExampleRSTFile.rst [hunk 1]
```diff
@@ -534,12 +534,14 @@ The empty cell is the second column in the first row of this table.
      :widths: 25 25 50
 
    * - Annotation Problem
+       (example annotation)
```
Comment (flynneva) [inline tests/source/ExampleRSTFile.rst:537]: @liran-funaro this is what is tripping up this builder I think

**Comment by flynneva** [comment]
I guess maybe not since the test is passing already...let me keep digging

File: sphinx_markdown_builder/contexts.py [hunk 2]
```diff
@@ -196,8 +199,8 @@ def make(self):
 class TableContext(SubContext):
     def __init__(self, params=SubContextParams()):
         super().__init__(params)
-        self.body: List[List[List[str]]] = []
-        self.headers: List[List[List[str]]] = []
+        self.body: List[List[List[str]]] = [[[]]]
+        self.headers: List[List[List[str]]] = [[[]]]
```
Comment (flynneva) [inline sphinx_markdown_builder/contexts.py:203]: @liran-funaro This last fix was needed as well for me at least.

**Comment by liran-funaro** [comment]
@flynneva Thank you for taking the time to investigate this issue.

I still don't see the error that was described on #3 when testing with your modifications.

Please note that the [last change](https://github.com/liran-funaro/sphinx-markdown-builder/pull/4#discussion_r1287917750) breaks other tests, so I suggest removing it from this PR.

Maybe you are running on a different version of sphinx? Can you share your version?

**Comment by liran-funaro** [comment]
@flynneva The changes you made in `ExampleRSTFile.rst` and `ExampleRSTFile.md` does not fail the test on the main branch on my local machine and in the CI.
Something in your environment may be different. We should find out what it is in order to resolve this bug.

**Comment by flynneva** [comment]
@liran-funaro no worries. I'm circling back to this today so ill try and update this PR either today or over the weekend 👍🏼
